Makefile

```diff
@@ -1,45 +1,64 @@
-JSONNET = https://github.com/google/jsonnet/releases/download/v0.17.0/jsonnet-bin-v0.17.0-linux.tar.gz
+JB = https://github.com/jsonnet-bundler/jsonnet-bundler/releases/latest/download/jb-linux-amd64
 BINDIR = bin
 TEMPLATESDIR = templates
 OUTPUTDIR = rendered
 ALLDIRS = $(BINDIR) $(OUTPUTDIR)
 SYNCER_IMG_TAG ?= quay.io/cloud-bulldozer/dittybopper-syncer:latest
 PLATFORM = linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
 
-# Get all templates at $(TEMPLATESDIR)
-TEMPLATES := $(wildcard $(TEMPLATESDIR)/**/*.jsonnet)
+ifeq ($(filter v2,$(MAKECMDGOALS)),v2)
+    # Set variables and instructions for v2
+  TEMPLATES := $(wildcard $(TEMPLATESDIR)/**/*-v2.jsonnet)
+	LIBRARY_PATH := $(TEMPLATESDIR)/vendor
+	JSONNET := https://github.com/cloud-bulldozer/utils/releases/download/v0.0.0/jsonnet-bin-v0.20.0-linux.tar.gz
```

Comment:
Same comments on this line as https://github.com/cloud-bulldozer/performance-dashboards/pull/80, other than that lgtm